src/features/unified-share-modal/flowTypes.js

```diff
@@ -208,7 +208,7 @@ export type justificationReasonType = {
 };
 export type getJustificationReasonsResponseType = {
     classificationLabelId: string,
-    options: Array<justificationReasonType>,
+    options: Array<justificationReasonType> | null,
```

Comment:
```suggestion
    options?: Array<justificationReasonType>,
```

sorry i used the wrong word, meant to say "optional", since you were getting an undefined not null.